README.md

```diff
@@ -62,6 +62,30 @@ export FLOW_PRIVATE_KEY=xxxxxxxxxxxx
 flow project deploy --network=testnet
 ```
 
+### (Optional) Heroku Deployment
```

Comment:
I think it'd be better to move this to an appendix lower down in the document, with a link at this point. Otherwise it breaks up the "Getting Started" section too much IMO